sci-mathematics/glucas/patches/glucas-2.9.0.patchset

```diff
@@ -0,0 +1,22 @@
+From 436ec67141e59325ecd135da8d9a3e98feffd1b3 Mon Sep 17 00:00:00 2001
+From: nishanth1232 <thumma.nishanth79@gmail.com>
+Date: Sat, 17 Nov 2018 18:30:26 +0000
+Subject: gsetup.c for glucas!
+
+
+diff --git a/gsetup.c b/gsetup.c
+index 7f32832..14a7354 100644
+--- a/gsetup.c
++++ b/gsetup.c
+@@ -39,7 +39,7 @@ void setup (int nonice)
+ #ifdef __sgi__
+     (void)schedctl(NDPRI, 0, NDPLOMIN);
+ #endif
+-#if !defined(macintosh) && !defined(pccompiler)
++#if !defined(macintosh) && !defined(pccompiler) && !defined(__HAIKU__)
+     (void)nice(40);
+ #endif
+   }
+-- 
+2.19.0
+ 
```

Comment:
Oops, the last line should not contain any character (other than the LineFeed (0x0a) character. Here there is white space character here, as we can see with this command:
```
tail -1 glucas-2.9.0.patchset | sed 's/ /#/g'
#
```
In case your text editor doe not permit to have en empty line at the end of the file you can fix this with:
```
sed -i -e "$ s/ //" glucas-2.9.0.patchset
```